tests/base/test_directives.py

```diff
@@ -21,19 +21,23 @@ class directivesValidation(unittest.TestCase):
     def test_validation_pass(self):
         betaest = directives.BetaEstimate_ByEig()
 
-        IRLS = directives.Update_IRLS(f_min_change=1e-4, minGNiter=3, beta_tol=1e-2)
+        IRLS = directives.Update_IRLS(f_min_change=1e-4, beta_tol=1e-2)
```

Comment:
Is there a reason why we don't want to replace the `Update_IRLS` for `UpdateIRLS` in this test file? If there's no particular reason, I would apply the following change:

```suggestion
        IRLS = directives.UpdateIRLS(f_min_change=1e-4, beta_tol=1e-2)
```

PS: remember to update the other appearances of `Update_IRLS` in this file if you accept this suggestion.

Reply:
Well I duplicated the tests in the previous PR, and it depends if you want to keep testing the old class before we deprecate it completely...

Reply:
I see. I agree we should keep the tests for `Update_IRLS` until we remove it. But I don't understand why we want to remove the `minGNiter` in these tests. I think we should keep the tests for `Update_IRLS` untouched for now to ensure the directive keeps working as it used to.

Reply:
Yep good point. It's probably just a mix up. Reverting.